nion/swift/model/Graphics.py

```diff
@@ -1706,7 +1749,16 @@ def draw(self, ctx: DrawingContextLike, ui_settings: UISettings.UISettings, mapp
     def label_position(self, mapping: CoordinateMappingLike, font_metrics: UISettings.FontMetrics, padding: float) -> typing.Optional[Geometry.FloatPoint]:
         p = Geometry.FloatPoint.make(mapping.map_point_image_norm_to_widget(self.position))
         return p + Geometry.FloatPoint(-self.cross_hair_size - font_metrics.height * 0.5 - padding * 2, 0.0)
+    
+    @property
+    def used_role(self) -> typing.Optional[str]:
```

Comment:
The used role only needs to be overridden for graphics that can only be used for a specific role. In this case, point and line may not have a specific role; their role is only set to 'mask' if the user chooses to 'add to mask' on the graphic. So... this method should not be implemented on either the point or the rectangle.

Reply:
Got it. I have removed this method from line and point graphics.